content/resources/chapter-8/building-blocks-6.tsx

```diff
@@ -22,7 +22,7 @@ const javascriptChallenge = {
   },
   defaultCode: `const getSubsidy = (height)=> {
   let halvings = BigInt(Math.floor(height / 210000));
-  if (halvings >= 64) {
+  if (halvings >= 33) {
```

Comment:
these changes are ok too, or we could probably even just remove them, its handled by Integer ayway